src/Components/Text/index.css

```diff
@@ -0,0 +1,6 @@
+.mainText {
```

Comment:
Same thing here, this is not following `SUIT CSS` naming conventions, the class name should be capitalized